packages/icons-react/src/createFromInfo.js

```diff
@@ -81,59 +81,63 @@ function iconToString(descriptor) {
 
 function createComponentFromInfo({ descriptor, moduleName }) {
   const source = `
-function ${moduleName}({ className, children, style, tabIndex, ...rest }) {
-  const { tabindex, ...props } = getAttributes({
-    ...rest,
-    tabindex: tabIndex,
-  });
-
-  if (className) {
-    props.className = className;
-  }
-
-  if (tabindex !== undefined && tabindex !== null) {
-    props.tabIndex = tabindex;
-  }
-
-  if (typeof style === 'object') {
-    props.style = {
-      ...defaultStyle,
-      ...style,
+    const ${moduleName} = React.forwardRef(({ className, children, style, tabIndex, ...rest }, ref) => {
+      const { tabindex, ...props } = getAttributes({
+        ...rest,
+        tabindex: tabIndex,
+      });
+
+      if (className) {
+        props.className = className;
+      }
+
+      if (tabindex !== undefined && tabindex !== null) {
+        props.tabIndex = tabindex;
+      }
+
+      if (typeof style === 'object') {
+        props.style = {
+          ...defaultStyle,
+          ...style,
+        };
+      } else {
+        props.style = defaultStyle;
+      }
+
+      if (ref) {
+        props.ref = ref;
+      }
+
+      return React.createElement(
+        'svg',
+        props,
+        children,
+        ${descriptor.content.map(iconToString).join(', ')}
+      );
+    })
```

Comment:
Don't think this matters since we format it anyways, but should we add a `;` here?